{"name": "test_mul_004", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_file_003.Test_Credence_003 object at 0x000001A1D1B6DC10>\n\n    def test_mul_004(self):\n        a = 3\n        b = 7\n        mul = a * b\n        print(\"Mul of a and b is :\" + str(mul))\n        if mul == 10:\n            assert True\n        else:\n>           assert False\nE           assert False\n\ntest_file_003.py:11: AssertionError"}, "attachments": [{"name": "stdout", "source": "1e04b444-d43c-457b-b818-1a379b775f4b-attachment.txt", "type": "text/plain"}], "start": 1690836142415, "stop": 1690836142415, "uuid": "302c3c17-40f9-40ad-8ff2-c37731318a3a", "historyId": "1da0023011657c42a38b1de108e70725", "testCaseId": "1da0023011657c42a38b1de108e70725", "fullName": "test_file_003.Test_Credence_003#test_mul_004", "labels": [{"name": "suite", "value": "test_file_003"}, {"name": "subSuite", "value": "Test_Credence_003"}, {"name": "host", "value": "Soumya"}, {"name": "thread", "value": "8316-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_file_003"}]}